lib/js/jscore.nim

```diff
@@ -123,3 +123,29 @@ since (1, 5):
       assert [1, 2, 3, 4, 5].copyWithin(0, 3) == @[4, 5, 3, 4, 5]
       assert [1, 2, 3, 4, 5].copyWithin(0, 3, 4) == @[4, 2, 3, 4, 5]
       assert [1, 2, 3, 4, 5].copyWithin(-2, -3, -1) == @[1, 2, 3, 3, 4]
+
+
+since (1, 7):
+  func isFrozen*(obj: auto): bool {.importjs: "Object.$1(#)".}
```

Comment:
Isn't it better to use some specific type like `JsRoot` or similar? IIRC there are some base JS types for the JS target